views/entry/success.tt

```diff
@@ -24,8 +24,14 @@ the same terms as Perl itself.  For a copy of the license, please reference
 [%- END -%]
 
 [%- IF poststatus.ml_string != ".edit.delete" -%]
+
 <p>[% extradata.security_ml | ml %]</p>
-<p>[% ".extradata.subject" | ml( subject => extradata.subject) %]</p>
+<p>[% ".extradata.subj" | ml %]
+    [% IF extradata.subject.length > 0 %]
+        [% ".extradata.subject.subject" | ml( subject => extradata.subject ) %]
```

Comment:
What was happening before was the subject was being passed to a single translation string as input.  But now we have split up the header ("The entry was posted with the following subject:") and the subject into separate strings, and the subject is either the entry subject text, or the text "(no subject)" - in the first case, passing it through the translation system gains us nothing.

If we want to go back to having everything combined into one translation string, though, so that we can reorder things on the fly, we'll need a slightly different approach from the one used here.  Something like:
```
[% IF extradata.subject.length > 0 %]
[% subject = extradata.subject | html %]
[% ELSE %]
[% subject = ".extradata.subject.no_subject" | ml %]
[% END %]
<p>[% ".extradata.subject" | ml( subject => subject) %]</p>
```